instruqt/proxy/track_scripts/setup-proxy-host

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+set -euxo pipefail
+
+echo "Running track setup script - proxy-host"
+epoc_start=`date +%s`
+
+# Wait for the Instruqt host bootstrap to finish.
+until [ -f /opt/instruqt/bootstrap/host-bootstrap-completed ]
+do
+    sleep 1
+done
+
+# package install
+yum install -y mlocate bind-utils
+
+cat <<EOF>> ~/.ssh/config
+Host isolated-host
+  StrictHostKeyChecking no
+EOF
+
```

Comment:
Do you run SSH from the proxy host in any of your challenges? You shouldn't need an SSH config if you don't.

Reply:
This is a leftover from my initial testing of name resolution and connectivity testing between multi-host labs, not needed now, will be useful for the HA lab later though